### PR TITLE
EG-4691: Removed references to 'production ready'. Fixed broken link. Added no…

### PR DESCRIPTION
**Pull request by taylorharvey-r3**
…te about admin credentials.

### CODE REVIEW

**Review by lemjclarke — DISMISSED** [review]

**Comment by lemjclarke** [comment]
Looks good, thanks @taylorharvey-r3.

File: content/en/docs/apps/bankinabox/user-interface/admin-ui-guide.md [hunk 1]
```diff
@@ -24,7 +24,16 @@ The default Admin user name and password are as follows:
...
 
-After entering your username and password, you will arrive at the home screen. From the left sidebar navigation, you can access different screens depending on your role (Admin or Customer).
+After entering the admin username and password, you will arrive at the home screen. From the left sidebar navigation, you can access different screens depending on your role (Admin or Customer).
+
+{{< note >}}
+As this application should not be used in production, you do not need to change the admin password after the initial login.
+
+However, if you wish to do so, you can edit the password in the database on this line: https://github.com/corda/bank-in-a-box/blob/release/bankinabox/1.0/clients/src/main/resources/db/changelog/user-schema-dml-v1.changelog.xml#L21
+
+The new password must be bcrypt encrypted.
```
Comment (ivanterziev-r3) [inline content/en/docs/apps/bankinabox/user-interface/admin-ui-guide.md:34]: Add a link to the bcrypt website (or standard page)

File: content/en/docs/apps/bankinabox/user-interface/admin-ui-guide.md [hunk 2]
```diff
@@ -24,7 +24,16 @@ The default Admin user name and password are as follows:
 * User name: admin
 * Password: password1!
 
-After entering your username and password, you will arrive at the home screen. From the left sidebar navigation, you can access different screens depending on your role (Admin or Customer).
+After entering the admin username and password, you will arrive at the home screen. From the left sidebar navigation, you can access different screens depending on your role (Admin or Customer).
+
+{{< note >}}
+As this application should not be used in production, you do not need to change the admin password after the initial login.
```
Comment (ivanterziev-r3) [inline content/en/docs/apps/bankinabox/user-interface/admin-ui-guide.md:30]: I thought in the original Jira ticket request there was a recommendation to even enforce password change, as per the security guidelines across R3? Can you pls check this again and if this is the case, amend the wording in this note to instruct the user to change their password after first login?

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM just a few things to check - added comments.

File: content/en/docs/apps/bankinabox/user-interface/admin-ui-guide.md (hunk 2 above)
Comment (taylorharvey-r3) [reply content/en/docs/apps/bankinabox/user-interface/admin-ui-guide.md:30]: There was a lot of discussion around this. I will double check with Florian and make changes if needed.

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**